StarDb/Calibrations/tpc/TpcAvgCurrent.C

```diff
@@ -1,7 +1,9 @@
+#include <cstring>
+
 TDataSet *CreateTable() {
   if (!gROOT->GetClass("St_TpcAvgCurrent")) return 0;
   TpcAvgCurrent_st row;
-  memset(row, 0, sizeof(TpcAvgCurrent_st));
+  std::memset(&row, 0, sizeof(TpcAvgCurrent_st));
```

Comment:
I would not be certain that root5 / CINT would work with std::memset vs memset... but seems to be passing tests.

Reply:
No complaints from root5 AFAICT